docs/learning/howto/acls/group-apikey.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Group/API Key Create User API Key"
+order: 600
+---
+
+# Group/API Key Create User API Ke
+
+## Use Case Description
```

Comment:
@wayfaringson - I think we should provide more background here.  Are users not allowed to create API keys by default?

Reply:
Because it is a system task, i don't believe they can do so without this (unless they are admin).

Reply:
this is a good add then